core/test/input/fd_pairs.cfg

```diff
@@ -76,4 +76,4 @@ bravais_lattice sc
 ### Number of basis cells along principal
 ### directions (a b c)
 n_basis_cells 2 2 1
-################# End Geometry ###################
\ No newline at end of file
+################# End Geometry ###################
```

Comment:
You shouldn't commit changes like this - use your IDE to look at the changes and make sure you avoid such things.